manifest/services.py

```diff
@@ -222,7 +252,12 @@ def make_paths_relative(self):
                             index_relative_path:
                         ]
 
-    def __reset_manifest_step(self, step_name: str):
+    def __reset_manifest_step(self, step_name: str) -> None:
+        """Reset manifest step
```

Comment:
An explanation on what it means to reset a manifest step would be of benefit for future developments